vowpalwabbit/core/tests/tutorial_test.cc

```diff
@@ -8,18 +8,21 @@
 
 TEST(Tutorial, CppSimulatorWithoutInteraction)
 {
-  auto ctr = simulator::_test_helper("--cb_explore_adf --quiet --epsilon 0.2 --random_seed 5");
+  auto ctr = simulator::_test_helper(
+      std::vector<std::string>{"--cb_explore_adf", "--quiet", "--epsilon=0.2", "--random_seed=5"});
   EXPECT_GT(ctr.back(), 0.37f);
   EXPECT_LT(ctr.back(), 0.49f);
 }
 
 TEST(Tutorial, CppSimulatorWithInteraction)
 {
-  auto ctr = simulator::_test_helper("--cb_explore_adf -q UA --quiet --epsilon 0.2 --random_seed 5");
+  auto ctr = simulator::_test_helper(
+      std::vector<std::string>{"--cb_explore_adf", "--quadratic=UA", "--quiet", "--epsilon=0.2", "--random_seed=5"});
```

Comment:
Shouldn't this be two strings?

Reply:
which one?

Reply:
--quadratic=UA
--epsilon=0.2
--random_seed=5

Reply:
Those are valid ways to pass the value of an option